gtfs.yml

```diff
@@ -147,10 +147,10 @@
     required: true
     inputType: GTFS_ID
     columnWidth: 6
-    helpContent: "The stop_id field contains an ID that uniquely identifies a stop or station. Multiple routes may use the same stop. The stop_id is dataset unique."
+    helpContent: "The stop_id field contains an ID that identifies a stop or station. Multiple routes may use the same stop. The stop_id is dataset unique."
```

Comment:
The PR description says the help text for `stop_code`, not `stop_id` was supposed be changed. Which one is correct?

Reply:
Good catch, should be stop_code. Updated in https://github.com/ibi-group/datatools-ui/pull/993/commits/e09c8a80627dc0ae8c3a8f28ab4dc6ed7777aa7f